src/db/tables/onerep_scans.ts

```diff
@@ -111,6 +111,39 @@ async function setOnerepScanResults(
   });
 }
 
+async function isOnerepScanResultForSubscriber(params: {
+  onerepScanResultId: number;
+  subscriberId: number;
+}): Promise<boolean> {
+  const result = await knex("onerep_scan_results")
+    .innerJoin(
+      "onerep_scans",
+      "onerep_scan_results.onerep_scan_id",
+      "onerep_scans.onerep_scan_id"
+    )
+    .innerJoin(
+      "subscribers",
+      "onerep_scans.onerep_profile_id",
+      "subscribers.onerep_profile_id"
+    )
+    .where(
+      "onerep_scan_results.onerep_scan_result_id",
+      params.onerepScanResultId
+    )
+    .andWhere("subscribers.id", params.subscriberId)
+    .first("onerep_scan_result_id");
+
+  return typeof result?.onerep_scan_result_id === "number";
+}
+
+async function markOnerepScanResultAsResolved(
+  onerepScanResultId: number
+): Promise<void> {
+  await knex("onerep_scan_results")
+    .update("manually_resolved", true)
+    .where("onerep_scan_result_id", onerepScanResultId);
```

Comment:
Q: does this need to set any `updated_at` columns, or is that done automatically via magic?

Reply:
Oh, good catch! It should, and actually TypeScript would have warned me about this if I had used the object syntax: ba55bebe9

Reply:
I think we could have these default to the current timestamp (or whatever resolution is appropriate) and then purposely not set this column when updating, but we should do whatever is clearest for someone reading the code.